### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by OleksiiLepetiukha**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <h1>Treloo</h1>
+        <nav class="navbar_top">
+            <ul>
+                <li><a href="">Review</a></li>
+                <li><a href="">Tips</a></li>
```
Comment (nadiein) [inline index.html]: Links shouldn't have empty href attr. If there are no real links, just put #

File: index.html [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                <li><a href="">Alerts</a></li>
+                <li><a href="">Blog</a></li>
+            </ul>
+            <button>Blog</button>
```
Comment (nadiein) [inline index.html]: Blog is part of the navigation but not separate button. In figma it jsut an example how should look like :hover state on navigation item.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+        </nav>
+        <div class="header_slogan">
+            <h2>Discover Your Life, Travel Where You Want</h2>
+            <p class="slogan">Would you explore natur paradise in the world, let’s find the best destination in world with us.</p>
+        </div>
+    </header>
+    <div class="search">
+        <span class="buttons">
+            <button><img src="Images/logos/Airplane Mode On.png" alt="logo">Flight</button>
```
Comment (nadiein) [inline index.html]: Buttons should be styled with css, but not with image.

Comment (nadiein) [inline index.html]: We should rename images for project like that:
- all letters lowercase
- backspaces replace with -

File: index.html [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+            <button><img src="Images/logos/Hotel Star.png" alt="logo">Hotel</button>
```
Comment (nadiein) [inline index.html]: Project directories should be in lowercase

File: index.html [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+        </span>
+        <div class="forms">
+            <form class="airports" action="" method="get">
```
Comment (nadiein) [inline index.html]: Do not leave empty attributes like action.
Use # symbol instead.
Do not use GET in forms. Unsafe for sensitive data.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+                <label for="living_from">Living From</label>
+                <input list="city_from" name="living_from" id="living_from">
+                <datalist id="city_from">
```
Comment (nadiein) [inline index.html:37]: We rarely use id as selector just in very specific cases. For styling in css as selector preferable class selectors.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+                    <option value="Dublin">
+                    <option value="London">
+                    <option value="Amsterdam">
+                    <option value="Berlin">
+                    <option value="Kharkiv">
+                </datalist>
+                <img src="Images/logos/gray plane.png" alt="logo">
```
Comment (nadiein) [inline index.html]: Png typically is heavy and need extra space. Better to use SVG icons.
There is library for that case - https://fontawesome.com/search

File: index.html [hunk 8]
```diff
@@ -0,0 +1,201 @@
...
+                <label for="going_to">Going to</label>
+                <input list="city_to" name="going_to" id="going_to">
```
Comment (nadiein) [inline index.html]: There are two option to use label in pair with input.
<label>
    input
</label>
When label contain input - easier to read and understand that input works with label together.
I'd recommend to use that option.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,201 @@
...
+            <div class="offer_block">
+                <img src="Images/image/Lona X.png" alt="Lona X">
+                <p class="date">22 DEC 2022</p>
+                <p><span class="relax">Relax</span> 7 Days,6 Nights</p>
+                <h4>Lona X</h4>
+                <p><b>340$</b>/Person</p>
+                <button>Book Now</button>
+            </div>
+        </div>
+        <div class="about_us">
```
Comment (nadiein) [inline index.html:95]: When you logically separate content on the page use instead section tag. It's semantically closer to what you want to accomplish.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,201 @@
...
+            <h3 class="about_us_header"> Travel Any Corner of The World With Us</h3>
+            <p class="about_us_text"> Would you explore nature paradise in the world, let’s find the best destination in world with us, Would you explore nature paradise in the world, let’s find the best destination in world with us. Would you explore nature paradise in the world, let’s find the best destination in world with us.
+
+                Would you explore nature paradise in the world, let’s find the best destination in world with us.</p>
+            <img class="about_us_img" src="Images/image/Travel blog.jpg" alt="Image">
```
Comment (nadiein) [inline index.html]: alt and title attributes should describe exact image and tell user what it should be. Usually it's fallback for cases when image didn't downloaded yet due to internet connection. Do not use long description. Not more then 5-20 symbols

File: index.html (hunk 1 above)
Comment (OleksiiLepetiukha) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (OleksiiLepetiukha) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (OleksiiLepetiukha) [reply index.html]: done

Comment (OleksiiLepetiukha) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (OleksiiLepetiukha) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (OleksiiLepetiukha) [reply index.html]: done

File: index.html (hunk 6 above)
Comment (OleksiiLepetiukha) [reply index.html:37]: done

File: index.html (hunk 8 above)
Comment (OleksiiLepetiukha) [reply index.html]: done

File: index.html (hunk 9 above)
Comment (OleksiiLepetiukha) [reply index.html:95]: I don't understand which unit you mean

File: index.html (hunk 10 above)
Comment (OleksiiLepetiukha) [reply index.html]: done

File: index.html [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+        <div class="promotion">
+            <p>Don’t Miss The 50% Discount if You register Today</p>
+            <button>Register Today</button>
+        </div>
+    </main>
+    <footer>
+        <p class="annotation">© 2022 Treloo LLC - All right reserved.</p>
+        <nav class="navbar_bottom">
+            <ul>
+                <li><a href="">Review</a></li>
```
Comment (nadiein) [inline index.html]: Check code one more time, if there are similar things that we discussed. Like # here

File: index.html (hunk 7 above)
Comment (OleksiiLepetiukha) [reply index.html]: done

File: index.html (hunk 11 above)
Comment (OleksiiLepetiukha) [reply index.html]: done

**Review by nadiein — APPROVED** [review]
Reviewed

**Review by nadiein — APPROVED** [review]
Reviewed changes

**Review by nadiein — APPROVED** [review]
Reviewed changes

**Pull request merged**